codegen/build_dbc.py

```diff
@@ -122,6 +80,12 @@ def get_key_by_val(dictionary, val):
 
     for msg_id, can_frame in can_messages.items():
         source = get_key_by_val(device_enum, can_frame.source)
+        # Checks for critical messages to make sure an ACK is added later
+        if can_frame.is_critical != None and can_frame.is_critical:
+            ACKABLE_MESSAGES.add(str(can_frame.source))
+        # Checks for signed messages
+        if can_frame.is_signed != None and can_frame.is_signed:
+            SIGNED_MESSAGES.append(str(can_frame.msg_name))
 
         def get_muxed_voltage_signal():
```

Comment:
I think the functions `get_muxed_voltage_signal()` and `get_muxed_solar_signal()` can be removed since we've changed the architecture of solar and bms. If we need a different way to do this we can extend this file later, but I'd like this file to be as minimal as possible for now.